experimental/packages/opentelemetry-instrumentation-http/src/http.ts

```diff
@@ -38,7 +38,6 @@ import {
 import type * as http from 'http';
 import type * as https from 'https';
 import { Socket } from 'net';
-import * as semver from 'semver';
```

Comment:
note that I actually use `semver` in #3625. I'll make sure to update the other PR if the other lands first